changelog/unreleased/enhancement-redesign-filestable

```diff
@@ -0,0 +1,7 @@
+Enhancement: Redesign FilesTable
```

Comment:
I'd say `Enhancement: Redesign files table related components` because the ODS doesn't know anything about a files table.

Reply:
yes sounds better